config/install/og.settings.yml

```diff
@@ -1 +1,6 @@
+features_ignore_og_fields: false
```

Comment:
well, in D7 it's nice that when exporting a content type to features, to have the possibility to exclude those fields, and declare them using `og_create_field()` in the profile. As far as I remember, we've done it for Drupal Commons. I could live without it :)